kong/plugins/response-ratelimiting/header_filter.lua

```diff
@@ -14,6 +14,9 @@ local _M = {}
 local function parse_header(header_value, limits)
   local increments = {}
   if header_value then
+    if type(header_value) == "table" then
+      header_value = table.concat(header_value, ",") -- combine per RFC 2616 section 4.2
+    end
```

Comment:
It really is too bad that we are concatenating this string with comma, only to be split again on the next instruction... How about not creating that string in the first place if not needed?